src/views/NetworkDetail.vue

```diff
@@ -332,18 +332,18 @@ import api from '@/api';
 import { App } from '@/types';
 
 export default Vue.extend({
-  name: 'GraphDetail',
+  name: 'NetworkDetail',
   filters: {
     appendArgs(url: string) {
-      return `${url}/?workspace=${this.workspace}&graph=${this.graph}`;
+      return `${url}/?workspace=${this.workspace}&graph=${this.network}`;
     },
   },
```

Comment:
The `appendArgs` filter doesn't appear to be used anywhere, so I think we can just delete this entire `filters` block.

Reply:
See 88c3a3f